pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -135,7 +135,17 @@ in buildPythonPackage rec {
     sha256 = "sha256-QcvoJqpZJXPSc9HLCJHetrp/hMESuC5kYl90d7Id0ZU=";
   };
 
-  patches = lib.optionals stdenv.isDarwin [
+  patches = [
+    (fetchpatch {
+      # Import distutils directly, not from setuptools
+      url = "https://github.com/pytorch/pytorch/commit/07767569c964552702bf374da753212eb9cde327.patch";
+      hash = "sha256-XNqKzIG3JYlQFeyng3UDkuEI9CC6mFD3aislM0H6W34=";
+      excludes = [
+        # we don't run the tests anyway
+        "test/run_test.py"
+      ];
+    })
+  ] ++ lib.optionals stdenv.isDarwin [
```

Comment:
```suggestion
```

Otherwise we will probably never notice if the patch would be out of date.

Reply:
That patch was darwin-only before, I don't care about it.